pkgs/by-name/po/posting/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
Seems like `python3.pkgs` causes issue with splicing, and the fix is to use `python3Packages` instead (and below)

- https://github.com/NixOS/nixpkgs/issues/211340